src/net_processing.cpp

```diff
@@ -582,6 +582,19 @@ class PeerManagerImpl final : public PeerManager
      */
     bool MaybeDiscourageAndDisconnect(CNode& pnode, Peer& peer);
 
+    /** Handle a transaction whose result was not MempoolAcceptResult::ResultType::VALID.
+     * @param[in]   was_rejected    Whether this tx has already been rejected before and is being
+     *                              reevaluated as an orphan. Used to avoid adding duplicate entries to
+     *                              vExtraTxnForCompact.
+     * Updates m_txrequest, m_recent_rejects, m_orphanage, and vExtraTxnForCompact. */
+    void ProcessInvalidTx(const CTransactionRef& tx, const TxValidationState& result, bool was_rejected, NodeId nodeid)
+        EXCLUSIVE_LOCKS_REQUIRED(!m_peer_mutex, g_msgproc_mutex, cs_main);
```

Comment:
```suggestion
    void ProcessInvalidTx(NodeId nodeid, const CTransactionRef& tx, const TxValidationState& result, bool was_rejected)
        EXCLUSIVE_LOCKS_REQUIRED(!m_peer_mutex, g_msgproc_mutex, cs_main);
```

nit: This is more inline with other method sigs in peerman.

Reply:
done